qui/models/qubes.py

```diff
@@ -46,12 +47,20 @@ def __call__(cls, *args, **kwargs):
         return cls._instances[cls]
 
 
+class PatientDBusConnection(dbus.bus.BusConnection, metaclass=_Singleton):
+    def __init__(self):
+        super().__init__()
+        timeout = 0x7fffffff / 1000  # crummy way to pass DBUS_TIMEOUT_INFINITE
```

Comment:
I'd still prefer to have some timeout here, even if default one is too low. For cases like the dbus service crashes at startup or so. Maybe 90s?

Reply:
I was hitting 52s for some reason, on my not even _super_ old notebook, with (supposedly) nothing going on in the background, so 90s sounds very tight to me. 300s?

Reply:
Ok, let it be 300